### ISSUE TITLE
object detection:  I changed the score converter to SOFTMAX,and class...loss: weighted_softmax

### ISSUE THREAD

**Issue by zyc4me** [tensorflow/models#5672]
but, why does the regulization losses continue increase during training and finally it is very big than other two loss?  however,the classification loss and loc loss is normal. AND when i use SIGMOID converter and weighted _sigmoid  classification loss,  i do not have this problom.
![image](https://user-images.githubusercontent.com/34511872/47916531-75ca4c00-dee1-11e8-90be-6f8d90e4213e.png)

and my config is as follow:

model {
  ssd {
    num_classes: 2
    image_resizer {
      fixed_shape_resizer {
        height: 256
        width: 256
      }
    }
    feature_extractor {
      type: "ssd_mobilenet_v1"
      depth_multiplier: 0.5
      min_depth: 16
      conv_hyperparams {
        regularizer {
          l2_regularizer {
            weight: 3.99999989895e-05
          }
        }
        initializer {
          truncated_normal_initializer {
            mean: 0.0
            stddev: 0.0299999993294
          }
        }
        activation: RELU_6
        batch_norm {
          decay: 0.999700009823
          center: true
          scale: true
          epsilon: 0.0010000000475
          train: true
        }
      }
      use_depthwise: true
    }
    box_coder {
      faster_rcnn_box_coder {
        y_scale: 10.0
        x_scale: 10.0
        height_scale: 5.0
        width_scale: 5.0
      }
    }
    matcher {
      argmax_matcher {
        matched_threshold: 0.5
        unmatched_threshold: 0.5
        ignore_thresholds: false
        negatives_lower_than_unmatched: true
        force_match_for_each_row: true
      }
    }
    similarity_calculator {
      iou_similarity {
      }
    }
    box_predictor {
      convolutional_box_predictor {
        conv_hyperparams {
          regularizer {
            l2_regularizer {
              weight: 3.99999989895e-05
            }
          }
          initializer {
            truncated_normal_initializer {
              mean: 0.0
              stddev: 0.0299999993294
            }
          }
          activation: RELU_6
          batch_norm {
            decay: 0.999700009823
            center: true
            scale: true
            epsilon: 0.0010000000475
            train: true
          }
        }
        min_depth: 0
        max_depth: 0
        num_layers_before_predictor: 0
        use_dropout: false
        dropout_keep_probability: 0.800000011921
        kernel_size: 3
        box_code_size: 4
        apply_sigmoid_to_scores: false
        use_depthwise: true
      }
    }
    anchor_generator {
      ssd_anchor_generator {
        num_layers: 5
        min_scale: 0.20000000298
        max_scale: 0.949999988079
        aspect_ratios: 1.0
        aspect_ratios: 0.10000000149
        aspect_ratios: 0.5
        aspect_ratios: 0.20000000298
        aspect_ratios: 0.333299994469
      }
    }
    post_processing {
      batch_non_max_suppression {
        score_threshold: 0.0100000118837
        iou_threshold: 0.5
        max_detections_per_class: 100
        max_total_detections: 100
      }
      score_converter: SOFTMAX
    }
    normalize_loss_by_num_matches: true
    loss {
      localization_loss {
        weighted_smooth_l1 {
        }
      }
      classification_loss {
        weighted_softmax {
        }
      }
      hard_example_miner {
        num_hard_examples: 3000
        iou_threshold: 0.990000009537
        loss_type: CLASSIFICATION
        max_negatives_per_positive: 3
        min_negatives_per_image: 3
      }
      classification_weight: 1.0
      localization_weight: 1.0
    }
  }
}
train_config {
  batch_size: 64
  data_augmentation_options {
    random_adjust_brightness {
    }
  }
  data_augmentation_options {
    random_adjust_contrast {
    }
  }
  data_augmentation_options {
    random_adjust_hue {
    }
  }
  data_augmentation_options {
    random_adjust_saturation {
    }
  }
  data_augmentation_options {
    random_distort_color {
    }
  }
  data_augmentation_options {
    ssd_random_crop {
    }
  }
  data_augmentation_options {
    random_horizontal_flip {
    }
  }
  optimizer {
    momentum_optimizer {
      learning_rate {
        manual_step_learning_rate {
          schedule {
            step: 1
            learning_rate: 0.0010000000475
          }
          schedule {
            step: 80000
            learning_rate: 0.000500000023749
          }
          schedule {
            step: 150000
            learning_rate: 0.000250000011874
          }
          schedule {
            step: 200000
            learning_rate: 0.000125000005937
          }
          schedule {
            step: 240000
            learning_rate: 1.24999996842e-05
          }
        }
      }
      momentum_optimizer_value: 0.899999976158
    }
  }
  fine_tune_checkpoint: "/home/zyc/tensorflow/coco_train/snapshot_coco_ssdlite_mobilenetv1_256_0.5x_5/model.ckpt-614942"
  from_detection_checkpoint: true
  num_steps: 2000000
}
train_input_reader {
  label_map_path: "/home/zyc/tensorflow/PersonDataMerge4/data/person_label_map.pbtxt"
  tf_record_input_reader {
    input_path: "/home/zyc/tensorflow/PersonDataMerge4/data/person_train.record"
  }
}
eval_config {
  num_examples: 1797
  use_moving_averages: false
  retain_original_images: true
}
eval_input_reader {
  label_map_path: "/home/zyc/tensorflow/PersonDataMerge4/data/person_label_map.pbtxt"
  shuffle: false
  num_readers: 1
  tf_record_input_reader {
    input_path: "/home/zyc/tensorflow/PersonDataMerge4/data/person_test.record"
  }
}

**Comment by shishichang** [comment]
@zyc4me  have you solve the problem?

**Comment by zyc4me** [comment]
> @zyc4me have you solve the problem?

use the newest  code

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by nilskk** [comment]
I have the same problem. Did you find a solution?
